geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/Runner.scala

```diff
@@ -20,11 +20,16 @@ import scala.util.control.NonFatal
 trait Runner {
 
   def name: String
+  def environmentErrorInfo(): Unit = {}
```

Comment:
we could make this return an Option[String] and control the output in the main Runner (with a logger)